components/src/core/list-item-tag/list-item-tag.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { TextProps, TextStyle, StyleProp, StyleSheet } from 'react-native';
+import { Theme, useTheme } from 'react-native-paper';
+import Color from 'color';
+import { white } from '@pxblue/colors';
+import { Overline } from '..';
+import { $DeepPartial } from '@callstack/react-theme-provider';
+
+export type ListItemTagProps = TextProps & {
```

Comment:
should this be TextProps, or TypographyProps (from our Typography components, which extend from TextProps)? If we extend from out TypographyProps, we could do things like:
```
<ListItemTag label="Hello" font={'bold' } fontSize={24}/>
```

and you'll get the theme and styles props for free

Reply:
Good idea... one more example of why all the types shall be exported